App.js

```diff
@@ -1,17 +1,37 @@
-import { createAppContainer } from 'react-navigation';
+import React from "react";
+import { createAppContainer, createSwitchNavigator } from 'react-navigation';
 import { createStackNavigator } from 'react-navigation-stack';
-import HomeScreen from "./src/screens/HomeScreen";
+import { createBottomTabNavigator } from 'react-navigation-tabs';
+import SigninScreen from "./src/screens/SigninScreen";
+import SignupScreen from "./src/screens/SignupScreen";
+import TrackCreateScreen from "./src/screens/TrackCreateScreen";
+import TrackDetailScreen from "./src/screens/TrackDetailScreen";
+import TrackListScreen from "./src/screens/TrackListScreen";
+import AccountScreen from "./src/screens/AccountScreen";
 
-const navigator = createStackNavigator(
-  {
-    Home: HomeScreen
-  },
-  {
-    initialRouteName: "Home",
-    defaultNavigationOptions: {
-      title: "App"
-    }
-  }
-);
+import { Provider as AuthProvider } from './src/context/AuthContext';
 
-export default createAppContainer(navigator);
+const switchNavigator = createSwitchNavigator({
```

Comment:
e de preferat pt o mai buna citire a codului ca rutele sa fie definita in alt fisier